proto/cmp/services/activity/v1alpha/activity_types.proto

```diff
@@ -0,0 +1,235 @@
+syntax = "proto3";
+
+// ### Activity Services
+//
+// The Activity services are used for both tickets and excursions
+
+package cmp.services.activity.v1alpha;
+
+import "cmp/types/v1alpha/description.proto";
+import "cmp/types/v1alpha/location.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/file.proto";
+import "cmp/types/v1alpha/datetime_range.proto";
+import "cmp/types/v1alpha/duration.proto";
+import "cmp/types/v1alpha/address.proto";
+import "cmp/types/v1alpha/phone.proto";
+import "cmp/types/v1alpha/email.proto";
+import "cmp/types/v1alpha/link.proto";
+import "cmp/types/v1alpha/product_code.proto";
+import "cmp/services/activity/v1alpha/search_result_types.proto";
+
+message ActivityExtendedInfo {
+    // Activity Product Details
+    ActivityProductInfoDetails activity_product_detail_response = 1;
+
+    // Units
+    repeated ActivityUnitInfo units = 2;
+
+    // Service Info
+    ActivityServiceInfo service_info = 3;
+
+    // Zones
+    repeated ActivityZoneInfo zones = 4;
+
+    // Descriptions with different languages
+    repeated cmp.types.v1alpha.LocalizedDescriptionSet descriptions = 5;
+
+    // Coordinates
+    cmp.types.v1alpha.Coordinate position =6;
+
+    // Activity Features
+    repeated ActivityFeature features = 7;
+
+    // Tags
+    repeated ActivityTag tags = 8;
+
+    // Languages
+    repeated cmp.types.v1alpha.Language languages = 9;
+
+    // Contact Info; Address, Email, Phone, Urls
+    ActivityContactInfo contact_info = 10;
+
+    // Images
+    repeated cmp.types.v1alpha.Image images = 11;
+
+    // Videos
+    repeated cmp.types.v1alpha.Video videos = 12;
+
+}
+
+message ActivityProductInfoDetails {
```

Comment:
```suggestion
message ActivityDetails {
```
How about we shorten the names a little? This is an activity detail message right? Do we need the "product info" part? 

We already have a very long names with the SDKs, lines in code are getting too long, uncomfortable to code.

Reply:
OK